jenkins_pipelines/environments/common/pipeline-pull-request.groovy

```diff
@@ -71,9 +71,9 @@ def run(params) {
                             echo "Checking ${params.builder_project}:${params.pull_request_number}"
                             sh "bash susemanager-utils/testing/automation/wait-for-builds.sh -u -a ${params.builder_api} -c $HOME/.oscrc -p ${params.builder_project}:${params.pull_request_number}"
                             if(params.publish_in_host) {
-                              echo "Publishing packages into http://${fqdn_jenkins_node}/workspace/${params.builder_project}:${params.pull_request_number}/openSUSE_Leap_15.2/x86_64"
-                              sh "bash -c \"rm -rf ${jenkins_workspace}/${params.builder_project}:${params.pull_request_number}/openSUSE_Leap_15.2/x86_64\""
-                              sh "bash susemanager-utils/testing/automation/publish-rpms.sh -p \"${params.builder_project}:${params.pull_request_number}\" -r openSUSE_Leap_15.2 -a x86_64 -d \"${jenkins_workspace}\""
+                              echo "Publishing packages into http://${fqdn_jenkins_node}/workspace/${params.builder_project}:${params.pull_request_number}/openSUSE_Leap_15.3/x86_64"
+                              sh "bash -c \"rm -rf ${jenkins_workspace}/${params.builder_project}:${params.pull_request_number}/openSUSE_Leap_15.3/x86_64\""
+                              sh "bash susemanager-utils/testing/automation/publish-rpms.sh -p \"${params.builder_project}:${params.pull_request_number}\" -r openSUSE_Leap_15.3 -a x86_64 -d \"${jenkins_workspace}\""
```

Comment:
We might want to replace the hardcoded  openSUSE_Leap_15.3 by `${params.build_repo}`

Reply:
Yes, good idea, the same julio is suggesting. Let me do it and I'll push it.

Reply:
done